framework/doc/content/js/init.js

```diff
@@ -13,11 +13,13 @@
             coverTrigger: false, // Displays dropdown below the button
             alignment: 'left' // Displays dropdown with edge aligned to the left of button
         });
+    }); // end of document ready
 
-   // $('.collapsible').collapsible({
-   //   onOpen: function(el) {var icons = $(el).find('i'); $(icons[0]).text('keyboard_arrow_up');},
-   //   onClose: function(el) {var icons = $(el).find('i'); $(icons[0]).text('keyboard_arrow_down');}
-      // });
-
-  }); // end of document ready
+    // Change video source for dark mode, see media.py RenderVideo
+    if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+        $('.moose-video source[data-dark-src]').each(function(){
+            var src = $(this).data('dark-src');
+            $(this).attr('src', src);
+        });
```

Comment:
@dschwen Any idea why this would work locally with a build --serve but not on the actual server?